developer-docs-site/docs/cli-tools/aptos-cli-tool/use-aptos-cli.md

```diff
@@ -770,7 +770,7 @@ In this case, see [Install the dependencies of Move Prover](install-aptos-cli#st
 
 In this example, we will use `DebugDemo` in [debug-move-example](https://github.com/aptos-labs/aptos-core/tree/main/crates/aptos/debug-move-example).
 
-First, you need to include Move nursery in your Move [toml file](https://github.com/aptos-labs/aptos-core/tree/main/crates/aptos/debug-move-example/Move.toml).
+First, you need to include Move nursery in your Move [.toml file](https://github.com/aptos-labs/aptos-core/tree/main/crates/aptos/debug-move-example/Move.toml).
```

Comment:
Isn't common vernacular to just call it a toml file, .toml == toml file, so wouldn't .toml file = toml file file?